src/components/OSCALComponentDefinitionControlImplementation.test.js

```diff
@@ -43,10 +43,14 @@ test("OSCALComponentDefinitionControlImplementation displays component parameter
       controls={controlsData}
     />
   );
-  const resultByProse = getByTextIncludingChildern(
-    "Does something with < control 1 / parameter 1 label > and < control 1 / parameter 2 label >"
+  const resultByProseBeginning = getByTextIncludingChildern(
+    "Does something with"
   );
-  expect(resultByProse).toBeVisible();
+  const param1 = within(resultByProseBeginning).getByText("< control 1 / parameter 1 label >");
+  const param2 = within(resultByProseBeginning).getByText("< control 1 / parameter 2 label >");
+  expect(resultByProseBeginning).toBeVisible();
+  expect(param1).toBeVisible();
+  expect(param2).toBeVisible();
```

Comment:
This is a solution for resolving the tests currently. With the way we're implementing our markup, we will need to grab the parameter values from within our main results body.

Reply:
I implemented the changes using regular expressions to check if they are in the document for this test